server/core/region_storage.go

```diff
@@ -31,14 +31,14 @@ var dirtyFlushTick = time.Second
 // RegionStorage is used to save regions.
 type RegionStorage struct {
 	*kv.LeveldbKV
-	mu           sync.RWMutex
-	batchRegions map[string]*metapb.Region
-	batchSize    int
-	cacheSize    int
-	flushRate    time.Duration
-	flushTime    time.Time
-	ctx          context.Context
-	cancel       context.CancelFunc
+	mu                  sync.RWMutex
+	batchRegions        map[string]*metapb.Region
+	batchSize           int
+	cacheSize           int
+	flushRate           time.Duration
+	flushTime           time.Time
+	regionStorageCtx    context.Context
```

Comment:
Why not keep the name `ctx` and `cancel`? It is a little verbose to use like this `regionStorage.regionStorageCtx`